server/ws.py

```diff
@@ -10,7 +10,7 @@
 # at https://www.sourcefabric.org/superdesk/license
 
 
-from settings import WS_HOST, WS_PORT, LOG_SERVER_ADDRESS, LOG_SERVER_PORT
+from settings import WS_HOST, WS_PORT, LOG_SERVER_ADDRESS, LOG_SERVER_PORT, BROKER_URL
```

Comment:
I'm not really sure. Again...this is something took for granted from superdesk because otherwise I was obtaining an error. Anyway, it something that celery needs.